e2e/fluentd-aggregator/fluentd_aggregator_test.go

```diff
@@ -401,3 +401,170 @@ func TestFluentdAggregator_ConfigChecks(t *testing.T) {
 		common.RequireNoError(t, rbacv1.AddToScheme(o.Scheme))
 	})
 }
+
+func TestFluentdAggregator_NamespaceLabel(t *testing.T) {
```

Comment:
This seems to be a duplicate of `TestFluentdAggregator_NamespaceLabel` in `e2e/fluentd-aggregator-namespacelabel/fluentd_aggregator_test.go`, is it intentional?

Reply:
no, thanks for pointing out, fixed in db86c9be